src/protocol/securemul.rs

```diff
@@ -18,15 +21,20 @@ pub struct DValue<F> {
 /// for use with replicated secret sharing over some field F.
 /// K. Chida, K. Hamada, D. Ikarashi, R. Kikuchi, and B. Pinkas. High-throughput secure AES computation. In WAHC@CCS 2018, pp. 13–24, 2018
 #[derive(Debug)]
-pub struct SecureMul<'a, G, S> {
+pub struct SecureMul<'a, G> {
     prss: &'a PrssSpace,
     gateway: &'a G,
-    step: S,
+    step: &'a UniqueStepId,
```

Comment:
we use step and step id interchangeably throughout the code for `UniqueStepId`. Is it going to cause some confusion and should we use `step_id` for it instead?

Reply:
We've used "step" far more frequently.  I think that it is probably better to change the name we use for the `Step` type instead.

Reply:
Should we also change `UniqueStepId` to `Step` since that is the more frequently used type? that would also align with the `step` var name